README.md

```
@@ -4,81 +4,102 @@
 ## Maintainers
 
-| GitHub | Role |
-|--------|------|
-| [<img src="https://github.com/Ryan-Millard.png" width="20" height="20" valign="middle" alt="Ryan-Millard avatar"> `@Ryan-Millard`](https://github.com/Ryan-Millard) | Lead Maintainer |
-| [<img src="https://github.com/krasner.png" width="20" height="20" valign="middle" alt="krasner avatar"> `@krasner`](https://github.com/krasner) | Core Maintainer |
+[![@Ryan-Millard](https://img.shields.io/badge/@Ryan--Millard-black?logo=github)](https://github.com/Ryan-Millard)
+[![@krasner](https://img.shields.io/badge/@krasner-black?logo=github)](https://github.com/krasner)
+[![@Prachi](https://img.shields.io/badge/@Prachi--Gupta2808-black?logo=github)](https://github.com/Prachi-Gupta2808)
+
+> [!NOTE]
+> #### Maintainers wanted
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)